### PR TITLE
Update Greek translation

### PR DESCRIPTION
**Pull request by alkisg**
Hi, I updated the Greek translation.
There were some missing strings, but most importantly the next/previous words were translated as previous/next.

### CODE REVIEW

**Comment by alkisg** [comment]
@squidfunk, I need a bit of help for the Greek search function.
My attempt above ^ fails with:

> WARNING  -  [18:45:12] "GET /assets/javascripts/lunr/min/lunr.el.min.js HTTP/1.1" code 404

If I change `"search.config.lang": "en",` instead of `el`, then I can search for English strings but not for Greek strings.

I might be able to send some patch for `el` to lunr in the following months, but for now, could we somehow have Greek search working even without stemming?

**Comment by alkisg** [comment]
Got it, with this additional commit Greek search works now (without stemming).

File: src/partials/languages/el.html [hunk 1]
```diff
@@ -34,9 +34,7 @@
   "meta.comments": "Σχόλια",
   "meta.source": "Πηγή",
   "nav.title": "Πλοήγηση",
-  "search.config.lang": "el",
-  "search.config.pipeline": "trimmer, stopWordFilter",
-  "search.config.separator": "[\s\-]+",
+  "search.config.pipeline": " ",
```
Comment (squidfunk) [inline src/partials/languages/el.html:37]: I think the `trimmer` and `stopWordFilter` should not be removed. My testing showed it also works with them.

**Review by squidfunk — CHANGES_REQUESTED** [review]

**Comment by alkisg** [comment]
By replacing:
> "search.config.pipeline": " ",

with:
> "search.config.pipeline": "trimmer, stopWordFilter",
  
Greek search stops working for me. I'll experiment more, but if you have any hints, they'll be most welcome. :)

**Comment by alkisg** [comment]
With this, it does work:

>   "search.config.pipeline": "stopWordFilter",

**Comment by alkisg** [comment]
As a test case, if I have the string:
"english ελληνικά"

then typing `en` in Search always matches the text, but typing `ελ` only works if `trimmer` has been removed.

**Review by squidfunk — APPROVED** [review]

**Pull request merged**